### PR TITLE
Portfolio Project : finish mobile version

### PR DESCRIPTION
**Pull request by jojo987N**
### Portfolio Project : finish mobile version
For the second milestone :

- Create the mobile website section
- Build the middle section of the mobile website
- Add "Works" section
- Add About myself section
- Begin Contact form section
- make sure there are no linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,28 +13,154 @@
...
+        <li class="tag">CSS</li>
+        <li class="tag">JavaScript</li>
+      </ul>
+      <div class="action">
+        <button>See project</button>
+      </div>
+    </div>
   </section>
+  <section class="about-myself headline">
+    <h1>About Myself</h1>
```
Comment (Strangeal) [inline index.html]: Please change the `h1` and make it an `h2` tag instead. You should have only one h1 tag as the main heading on the page.

File: index.html [hunk 2]
```diff
@@ -13,28 +13,154 @@
...
+  <section class="Works">
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio.png" alt="">
+      <h2>Tonic</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>2015</li>
```
Comment (Strangeal) [inline index.html]: Great 😊 work so far. Please include text in the `alt`. It's important to always include `alt` text for all images. The alt text improves the accessibility of a website for screen readers and also shows up on the screen when your images fail to load.

File: index.html [hunk 3]
```diff
@@ -13,28 +13,154 @@
...
+      </div>
+    </div>
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio1.png" alt="">
+      <h2>Multi-Post Stories</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
```
Comment (Strangeal) [inline index.html]: Please refer back to the previous comment above 👆 about including text in the `alt`.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @jojo987N, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights
Impressive design 😊🥳🍻

## Required Changes ♻️
- [ ] Please make sure to read the comment I gave below about the `alt` and the importance of using it and include a valid text for each of your images.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -13,28 +13,154 @@
...
-      </ul>
-     
+
+    <ul class="media-icon">
+      <li><a href="#"><img src="img/twitter.png"></a></li>
+      <li><a href="#"><img src="img/Linkedin.png"></a></li>
+      <li><a href="#"><img src="img/m.png"></a></li>
+      <li><a href="#"><img src="img/github.png"></a></li>
+      <li><a href="#"><img src="img/fingers.png"></a></li>
+    </ul>
```
Comment (LionRouge1) [inline index.html:35]: [OPTIONAL] 
- [ ] Please remember to always add `alt` attribute to an `img` tag, as mentioned by the previous reviewer it's important to always include `alt` text for all images. The alt text improves the accessibility of a website for screen readers and also shows up on the screen when your images fail to load.

File: index.html [hunk 5]
```diff
@@ -13,28 +13,154 @@
...
+      contact me.</p>
+    <div class="third-text">LET’S CONNECT</div>
+
+    <ul class="media-icon">
+      <li><a href="#"><img src="img/twitter.png"></a></li>
+      <li><a href="#"><img src="img/Linkedin.png"></a></li>
+      <li><a href="#"><img src="img/m.png"></a></li>
+      <li><a href="#"><img src="img/github.png"></a></li>
+      <li><a href="#"><img src="img/fingers.png"></a></li>
+    </ul>
```
Comment (LionRouge1) [inline index.html:133]: [OPTIONAL]
- [ ] Same here too :point_up_2:

**Review by LionRouge1 — APPROVED** [review]
Hi @jojo987N , 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**